packages/ts-transforms/src/loader/rules-loader.ts

```diff
@@ -26,14 +37,34 @@ export default class RulesLoader {
         return JSON.parse(strConfig);
     }
 
-    private async fileLoader(ruleFile: string): Promise<OperationConfigInput[]> {
+    private async dataLoader(ruleFile: string|false, notifications?: string): Promise<OperationConfigInput[]> {
         const parseConfig = this.parseConfig.bind(this);
         const results: OperationConfigInput[] = [];
         const errorResults: string[] = [];
         let hasError = false;
 
+        let inputStream;
+
+        if (ruleFile) {
+            inputStream = fs.createReadStream(ruleFile);
+        } else if (notifications != null && notifications.length > 0) {
+            const notificationsRules = [notifications];
+
+            inputStream = new Readable({
+                read() {
+                    const rule = notificationsRules.pop();
+                    if (!rule) {
+                        this.push(null);
```

Comment:
You should return here?
```suggestion
                        this.push(null);
                        return;
```